Makefile

```diff
@@ -8,8 +8,17 @@ export PROJECT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 export ADI_DIR:=${PROJECT_DIR}/adi
 export ROOT_DIR:=$(shell dirname "${PROJECT_DIR}")
 
+.PHONY: build test
+
 build:
-	bash ${ADI_DIR}/build.sh
+	echo "${VERSION}" > VERSION.txt
+	pip install -r ${PROJECT_DIR}/requirements/test.txt
 
 test:
 	bash ${ADI_DIR}/test.sh
+
+release:
+	bash ${ADI_DIR}/release.sh
+
+version:
+	echo "1.0.${BUILD_IDENTIFIER}"
```

Comment:
It's interesting yes. I prefer it that way just cos I use pyenv and i know not everyone uses pyenv. For example if i'm using py3.X locally with pyenv, `virtualenv` command is unavailable so this can't do the magic I want it to - at least not for me.

But I also understand that it may be desirable to do *everything*. It's a question of where to draw the line of ownership I suppose.